omnibus/files/private-chef-cookbooks/private-chef/libraries/helper.rb

```diff
@@ -84,9 +84,13 @@ def elastic_search_major_version
       end
       raise "Unable to interrogate elasticsearch server - HTTP#{res.code}:\n#{res.body}" if res.code.to_i >= 400
       # This can raise exceptions if the response does not match the format we expect.
-      version = JSON.parse(res.body)['version']['number'].split('.').first.to_i
-      raise "Unsupported elasticsearch version of #{version}. There is current support for the major versions of 2 and 5." if version != 5 && version != 2
-      version
+      begin
+        version = JSON.parse(res.body)['version']['number'].split('.').first.to_i
+        raise "Unsupported elasticsearch version of #{version}. There is current support for the major versions of 2 and 5." if version != 5 && version != 2
+        version
+      rescue => e
```

Comment:
It was attempting to decorate the JSON exceptions with more context.  I restructured it to be more explicit.